src/h5web/visualizations/containers/NxImageContainer.tsx

```diff
@@ -76,7 +82,11 @@ function NxImageContainer(props: VisContainerProps): ReactElement {
       />
       <MappedHeatmapVis
         value={datasetValues[signalName]}
-        title={getDatasetLabel(signalDataset, signalName)}
+        title={
+          datasetValues.title && typeof datasetValues.title === 'string'
```

Comment:
You don't need the first part of the condition, since `typeof undefined === "undefined"`